### PR TITLE
Revise backup recovery database page

### PR DESCRIPTION
**Pull request by Ayato77**
# Description

- The process for backup and restore is updated and tested. (The old instruction do not consider hypertables.)
- For `umh_v2`, it mentions which username, password should be used instead of `factoryinsight`.
- This guide is used: https://docs.timescale.com/self-hosted/latest/migration/schema-then-data/

## Related issues

close united-manufacturing-hub/MgmtIssues/issues/1338

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md [hunk 1]
```diff
@@ -57,98 +64,145 @@ machine:
...
+1. Make sure that your device is connected to server via SSH and run the following command:
+
+   {{< include "open-database-shell" >}}
 
-1. Drop the existing database:
+2. Drop the existing database:
 
    ```sql
-   DROP DATABASE factoryinsight;
+   DROP DATABASE factoryinsight WITH (FORCE);
```
Comment (Scarjit) [inline umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md:141]: We might want to not do this, but instead include instructions on how to turn off kafkatopostgresql & kafkatopostgresqlv2 while dropping, as they might be writing actual data !

Comment (Ayato77) [reply umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md]: kafkatopostgresql, kafkatopostgresqlv2, and factoryinsight are disabled while dropping now.

**Comment by JeremyTheocharis** [comment]
@Scarjit Is your comment resolved now?

**Comment by Scarjit** [comment]
yes

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**